redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -465,29 +465,33 @@ object Input {
       Chunk.single(encodeString(data.stringify))
   }
 
-  case object GetExPersistInput extends Input[(String, Boolean)] {
-    override private[redis] def encode(data: (String, Boolean)): Chunk[RespValue.BulkString] =
-      if (data._2) Chunk(encodeString(data._1), encodeString("PERSIST")) else Chunk(encodeString(data._1))
-  }
-  case object GetExInput extends Input[(String, Expire, Duration)] {
-    override private[redis] def encode(data: (String, Expire, Duration)): Chunk[RespValue.BulkString] =
+  case class GetExPersistInput[K: Schema]() extends Input[(K, Boolean)] {
+    override private[redis] def encode(data: (K, Boolean))(implicit codec: Codec): Chunk[RespValue.BulkString] =
```

Comment:
Let's remove `override private`. I noticed that some of the recently added inputs are using it, but please remove it everywhere together with the ones from this PR.